src/components/Calculator.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+// eslint-disable-next-line react/prefer-stateless-function
```

Comment:
- [ ] @Christabelle143, let's fix issues flagged by Linters tools instead of disabling them. Linters are great tools that analyze source code to flag programming errors, bugs, and stylistic errors.